libatalk/iniparser/dictionary.c

```diff
@@ -34,31 +34,27 @@ DEALINGS IN THE SOFTWARE.
 /*--------------------------------------------------------------------------*/
 
 /*---------------------------------------------------------------------------
-   								Includes
+                                                                Includes
  ---------------------------------------------------------------------------*/
-#ifdef HAVE_CONFIG_H
 #include "config.h"
-#endif /* HAVE_CONFIG_H */
```

Comment:
Any particular reason for removing this ifdef here? I think HAVE_CONFIG_H is still used in most other places in the codebase.